db/migrations/0015.add_mivideo_creation_tables.py

```diff
@@ -0,0 +1,28 @@
+'''
+Migration for MiVideo media started hourly table
+'''
+
+from yoyo import step
+
+__depends__ = {'0012.add_mivideo_usage_table'}
+
+
+steps = [
+    step('''
+        CREATE TABLE IF NOT EXISTS mivideo_media_created (
+            id VARCHAR(20) NOT NULL UNIQUE,
+            created_at DATETIME NOT NULL,
+            name VARCHAR(200) NOT NULL,
+            duration BIGINT NOT NULL,
+            PRIMARY KEY (id)
+        ) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4;
+    '''),
+    step('''
+        CREATE TABLE IF NOT EXISTS mivideo_media_courses (
```

Comment:
Also, did you give any thought to an auto-incremented PK instead of the compound key? Maybe there was a concern related to the `_queryRunner` part of this?

Reply:
Name: I could possibly see naming it `mivideo_media_placement` or `_relationship`.  However, it **_is_** supposed to show the relationship between the `mivideo_media_creation` table and the `course` table.  Also, if we are worried about the name of the table, shouldn't we worry about the name of the `course_id` column?

Key: I wanted the table's `media_id`, `course_id`, `in_context` triple to be unique.  I was under the impression that in MySQL, making the columns part of a compound key was the right (or a good) way to do that.  If we can make MySQL enforce that uniqueness without the columns being the key, then I could go along with making the key an auto-incremented ID instead.  If you can point me to the documentation that shows how to do that, I could do that.

Reply:
Name: What do you mean about being worried about `course_id`? That's an unenforced foreign key; all other foreign keys to the `course` table use `course_id`. Just in general, I think it's useful to understand what you are modeling with a table; what does one record represent? For example, I would prefer `enrollment` over `course_user`. I'm not insisting on you changing this; you can change or resolve based on what you think would be least confusing to others.

Reply:
I didn't think you were insisting on changing this.  I was trying to understand the logic of your argument.  I think for now, we'll leave this as is.  I'll open an issue to think about it again later.

I hope I know enough, but I'm always happy to learn more.  I had wanted to use an autoincrementing ID key, but I couldn't find a way to do that and keep these triples unique.